lib/SIL/TypeLowering.cpp

```diff
@@ -1472,9 +1472,13 @@ CanType TypeConverter::getLoweredRValueType(AbstractionPattern origType,
     auto extInfo = substFnType->getExtInfo();
     if (getSILFunctionLanguage(extInfo.getSILRepresentation())
           == SILFunctionLanguage::C) {
+      auto bridging = extInfo.getSILRepresentation()
+                        == SILFunctionTypeRepresentation::CFunctionPointer
+          ? Bridgeability::None : Bridgeability::Full;
```

Comment:
Nitpick: it might be better to reverse this check so that "Full" is the restricted case.